zerver/tests/fixtures/slack_message_conversion.json

```diff
@@ -84,6 +94,16 @@
       "name": "italic_and_strike_conversion",
       "input": "_~italic~_ and ~_strike_~",
       "conversion_output": "*~~italic~~* and ~~*strike*~~"
+    },
+    {
+      "name": "unsupported_bold_test",
+      "input": "*foo* *bar*",
```

Comment:
We couldn't capture this bold string because the `SLACK_BOLD_REGEX` have a capture group that matches for a character before asterisk "*"  
```python
SLACK_BOLD_REGEX = r"""
                    (\n|^|[ -(]|[+-/]|[:-?]|\{|\[|\_|\||\^|~)
                    ...
```
In this case, both `**foo**` and `*bar*` shares that one character, so one bold string won't get matched because that starting/ending character was used to match the other string. 

There's probably a way to support this case, but I don't think the potential complexity would be worth it because in practice Slack export shouldn't produce this kind of string, if two words got bolded separately like this example, then when exported it will just turn up like this: `*foo bar*`.